pipelinewise/cli/__init__.py

```diff
@@ -19,7 +19,8 @@
 
 __version__ = get_distribution('pipelinewise').version
 USER_HOME = os.path.expanduser('~')
-CONFIG_DIR = os.path.join(USER_HOME, '.pipelinewise')
+DEFAULT_CONFIG_DIR = os.path.join(USER_HOME, '.pipelinewise')
+CONFIG_DIR = os.environ.get('PIPELINEWISE_CONFIG_DIRECTORY', DEFAULT_CONFIG_DIR)
```

Comment:
Look good to me, can you add this new env var to the documentation, somewhere under the installation section should suffice:
![image](https://user-images.githubusercontent.com/54845154/154063229-5e3f7d0e-8605-4b84-9050-de156d8ca31c.png)

Reply:
I added a note under the CLI section. Will that do?